pool/web/handler/admin_custom_fields.ml

```diff
@@ -0,0 +1,159 @@
+module HttpUtils = Http_utils
+module Message = HttpUtils.Message
+
+let create_layout req = General.create_tenant_layout `Admin req
+
+let boolean_fields =
+  Custom_field.boolean_fields |> CCList.map Pool_common.Message.Field.show
+;;
+
+let find_assocs_in_urlencoded urlencoded field encoder =
+  let field = Pool_common.Message.Field.show field in
+  CCList.filter_map
+    (fun (key, values) ->
+      let group, id = CCString.take_drop (CCString.length field) key in
+      match CCString.equal field group with
+      | false -> None
+      | true ->
+        let key =
+          let open CCOption in
+          id
+          |> CCString.chop_prefix ~pre:"["
+          >>= CCString.chop_suffix ~suf:"]"
+          >>= fun l -> l |> encoder
```

Comment:
- `>>= encoder`
- if "value" is defined before this "match with" statement, could it directly be included?